src/android/Order.java

```diff
@@ -39,6 +40,8 @@ public class Order {
 	public String submit_time_display;
 	public int time;
 	public String courier_due;
+	public int delivery_id;
```

Comment:
Third party IDs should always be handled as strings, since we don't know if their format will change, and it gives you provider independence (like POS IDs)